project.clj

```diff
@@ -24,7 +24,7 @@
 
   :test-paths ["test/common"] ;; See `test-clj` and `test-cljs` profiles below.
 
-  :profiles {:provided {:dependencies [[org.clojure/clojure "1.7.0"]]}
+  :profiles {:provided {:dependencies [[org.clojure/clojure "1.9.0-alpha10"]]}
```

Comment:
I don't think we should depend on 1.9. We need a better way to use the spec functions.